eng/CredScanSuppression.json

```diff
@@ -1,56 +1,43 @@
 {
   "tool": "Credential Scanner",
   "suppressions": [
-    {
-      "hash": "5ETIzLYe9aa9Xx5512Uy6gJiCZ/CL2QzS3nDfN51nz8=",
-      "_justification": "Secret used by test code"
-    },
     {
       "placeholder": [
-        "SharedKey fakestorageaccount:pass123",
-        "1cb763e7c4b9a3c4811c4560af6827c2",
-        "009fea5280ed80b780d40e4d0f6012dee17b86c49ef4eee29ad5e59008fc551e",
-        "AJ/qUoDtgLeA1A5ND2AS3uF7hsSe9O7imtXlkAj8VR4=",
-        "79f43c3b9c23fbdf293bdba80dab38ba194fc1da79187091b3d3760dad840bf8",
-        "efQ8O5wj+98pO9uoDas4uhlPwdp5GHCRs9N2Da2EC/g=",
         "`$`{NPM_TOKEN`}",
-        "azure_client_secret&scope=https%3A%2F%2Fsanitized%2F",
-        "azure_client_secret&scope=https%3A%2F%2Fattest.azure.net%2F.default",
-        "azure_client_secret&scope=https%3A%2F%2Fcognitiveservices.azure.com%2F.default",
-        "SecretBaseUnion",
-        "SecureString",
-        "azure_client_secret&scope=https%3A%2F%2Feventhubs.azure.net%2F.default",
-        "azure_client_secret&scope=https%3A%2F%2Fcognitiveservices.azure.com%2F.default",
-        "azure_client_secret&scope=https%3A%2F%2Fdigitaltwins.azure.net%2F.default",
-        "azure_client_secret",
-        "p@55wOrd",
-        "password",
-        "nodesdk",
-        "liilef#$DdRGSa_ewkjh",
-        "kt#_gahr!@aGERDXA",
-        "YmFzZTY0X3BsYWNlaG9sZGVy",
-        "SecretPlaceholder",
+        "password: fakeTestPassword",
+        "password: fakeTestSecretPlaceholder",
```

Comment:
If we have placeholder in value, then credscan will ignore the error. 
Do we still report the error without suppressing this?

Reply:
Rename to have `-Placeholder` suffix.  Removed these two and trying now.